robot_sort/robot_sort.py

```diff
@@ -91,13 +91,220 @@ def light_is_on(self):
+                    self.move_right()
+                
+                elif self.compare_item() == 0:
+                    self.set_light_on()
+                    self.move_left()
+                    self.swap_item() #drop item
+                    self.go_to_beginning()
+                    self.swap_item()
+                    self.move_right()
+
+            if self.compare_item() == -1:
+                # print("\n-1 so curent item smaller than compare so go left : swap : go right : pick up")
+                # print("go left")
+                self.move_left()
+                # print("Position:",self._position, "holding:",self._item)
+
+                # print("drop first item")
+                self.swap_item()
+                # print("Position:",self._position, "holding:",self._item)
+
+                # print("move right")
+                self.move_right()
+                # print("Position:",self._position, "holding:",self._item)
+
+                # print("pick up larger item")
+                self.swap_item()
+                # print("Position:",self._position, "holding:",self._item)
+
+                # print("move right")
+                self.move_right()
+                # print("Position:",self._position, "holding:",self._item)
+                # print(self._list)
+
+            if self.compare_item() == 0:
+                # print("Item same: move right")
+                self.move_left() 
+                self.swap_item()
+                self.move_right()
+                self.swap_item()
+                self.move_right()
+                # print("Position:",self._position)
+
+
+            if self.compare_item() == 1:
+                self.set_light_off() # turn light off if there is work to do
+                # print("\n1 so curent item bigger than compare: swap : go right: turn light off")
+                # print("Position:",self._position, "holding:",self._item)
+
+                # print("swap")
+                self.swap_item() # Grab for smaller item
+                # print("Position:",self._position, "holding:",self._item)
+
+                # print("go left")
+                self.move_left()
+                # print("Position:",self._position, "holding:",self._item)
+
+                # print("drop item")
+                self.swap_item()
+                # print("Position:",self._position, "holding:",self._item)
+
+                # print("go right") 
+                self.move_right()
+                # print("Position:",self._position, "holding:",self._item)
+
+                # print("pick up larger item")
+                self.swap_item() # grab bigger item
+                # print("Position:",self._position, "holding:",self._item)
+
+                # print("move right")
+                self.move_right()
+                # print("Position:",self._position, "holding:",self._item)
+                # print(self._list)
+
+
+
+        print("End of the line")
+        
+        return self._list
```

Comment:
Great job! Think about how you might be able to reduce the number of lines of code...